vignettes/esApply.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "esApply Introduction"
+author: "R. Gentleman, V. Carey, M. Morgan, S. Falcon"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: Biobase
+vignette: >
+  %\VignetteIndexEntry{esApply Introduction}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+# A note on `esApply` {#a-note-on-esapply .unnumbered}
+
+`ExpressionSet`s are complex objects. `exprs(ExpressionSet)` produces $G \times N$, where $G$ is the number of genes on a chip and $N$ is the number of tissues analyzed, and `pData(ExpressionSet)` produces $N \times p$, where $p$ is the number of phenotypic or demographic, etc., variables collected.
+
+Abstractly, we are often interested in evaluating functions $f(y;x)$ where $y$ is an $N$-vector of expression results for a specific gene and $x$ is an $N$-dimensional structure, coordinated with $y$, that distinguishes elements of $y$ for processing in the function $f$. A basic problem is to guarantee that the $j$th element of $y$ is correctly associated with the $j$th component of $x$.
+
+As an example, let's consider `sample.ExpressionSet`, which is an *ExpressionSet* supplied with Biobase. We will print a little report, then the first $N$-vector of gene expressions and some covariate data:
+
+```{r message=FALSE, echo=FALSE}
+library(Biobase)
+data(sample.ExpressionSet)
+
```

Comment:
I don't know whether it has consequences or not, but the blank line at the end of the code chunk should be avoided.